### PR TITLE
Added argument to disable association removal from CMR during sync

### PR DESCRIPTION
**Pull request by Phoeneix**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -17,14 +18,26 @@ if [[ $env == "sit" || $env == "uat" ]]; then
   jq --arg a "https://harmony.uat.earthdata.nasa.gov" '.URL.URLValue = $a' cmr/cmr.json > cmr/cmr.json.tmp && mv cmr/cmr.json.tmp cmr/cmr.json
 fi
 
-if [[ "${LAUNCHPAD_TOKEN_SIT}" ]] && [[ $env == "sit" ]]; then
-  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p ${provider} -e ${env} -t $LAUNCHPAD_TOKEN_SIT -to $timeout
-elif [[ "${LAUNCHPAD_TOKEN_UAT}" ]] && [[ $env == "uat" ]]; then
-  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p ${provider} -e ${env} -t $LAUNCHPAD_TOKEN_UAT -to $timeout
-elif [[ "${LAUNCHPAD_TOKEN_OPS}" ]] && [[ $env == "ops" ]]; then
-  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p ${provider} -e ${env} -t $LAUNCHPAD_TOKEN_OPS -to $timeout
+if disable_removal == 'true'; then
```
Comment (sliu008) [inline entrypoint.sh:21]: can't we just pass in disable_removal during the command argument instead of making an if else

Comment (Phoeneix) [reply entrypoint.sh:21]: The -r input is a bool, that gets value if the argument is there or not, just like the -d for debug.

File: podaac/umms_updater/util/create_assoc.py [hunk 2]
```diff
@@ -103,7 +105,8 @@ def sync_association(cmr_env, concept_id, current_token, association, timeout=30
                 LOGGER.info("Failed add association: concept_id being associated "
                             "may not be valid: %s", assoc_concept_id)
 
-        for assoc_concept_id in remove:
+        LOGGER.info("Allow association removal: %s", remove_association)
+        for assoc_concept_id in remove and remove_association:
```
Comment (sliu008) [inline podaac/umms_updater/util/create_assoc.py:109]: this isn't valid python to loop and if in one statement

Comment (sliu008) [reply podaac/umms_updater/util/create_assoc.py:109]: use an if statement and then move the line remove = list(set(current) - set(new)) into if statement, so we only get the remove list if we want to remove

Comment (Phoeneix) [reply podaac/umms_updater/util/create_assoc.py]: changed

**Review by jamesfwood — APPROVED** [review]

**Pull request merged**